src/frontend/hooks/useHentKlagebehandlinger.ts

```diff
@@ -4,12 +4,14 @@ import { useHttp } from '@navikt/familie-http';
 
 import { hentKlagebehandlinger } from '../api/hentKlagebehandlinger';
 
-export const KLAGEBEHANDLINGER_QUERY_KEY_PREFIX = 'klagebehandlinger';
+export const HentKlagebehandlingerQueryKeyFactory = {
+    fagsak: (fagsakId: number | undefined) => ['klagebehandlinger', fagsakId],
```

Comment:
Trenger egentlig denne å ta i mot `undefined`?


```suggestion
    fagsak: (fagsakId: number) => ['klagebehandlinger', fagsakId],
```